cluster/src/main/scala/org/apache/pekko/cluster/SeedNodeProcess.scala

```diff
@@ -65,8 +80,14 @@ private[cluster] abstract class SeedNodeProcess(joinConfigCompatChecker: JoinCon
     val configToValidate =
       JoinConfigCompatChecker.filterWithKeys(requiredNonSensitiveKeys, context.system.settings.config)
 
+    val adjustedConfig = if (needsAkkaConfig)
+      ConfigUtil.addAkkaConfig(configToValidate, akkaVersion)
+    else configToValidate
+
+    logInfo("init-join config = " + adjustedConfig)
```

Comment:
@pjfanning `init-join` seems to be devoid of relevant/important info here, should this be updated?

Reply:
removed the log statement - was useful when debugging but not useful now